submissions/shawn-sean-anthony/scripts/article.js

```diff
@@ -0,0 +1,60 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
```

Comment:
Looking good! We'll soon cover a dynamic way to obtain all properties without having to explicitly list them all out.